merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/ResponseSerializers.java

```diff
@@ -101,8 +103,7 @@ public static JsonValue serializeStringList(final List<String> elements) {
   }
 
   public static JsonValue serializeArgument(final SerializedValue parameter) {
-    if (parameter == null) return JsonValue.NULL;
```

Comment:
Looks like we lost this null check. Not sure why it was there in the first place, but we should probably keep it.